docs/rackhd/ubuntu_source_installation.rst

```diff
@@ -1,6 +1,102 @@
 Source Installation on Ubuntu
 ---------------------------------
 
+Prerequisites
+~~~~~~~~~~~~~
+NICs
+^^^^
+
+
+**Ubuntu 14.04**
+
+Start with an Ubuntu trusty(14.04) instance with 2 nics:
+
+* eth0 for the 'public' network - providing access to RackHD APIs, and providing
+  routed (layer3) access to out of band network for machines under management
+
+* eth1 for dhcp/pxe to boot/configure the machines
+
+edit the network:
+
+* eth0 - assign IP address as appropriate for the environment, or you can use DHCP
+
+* eth1 static ( 172.31.128.0/22 )
+
+  this is the 'default'. it can be changed, but more than one file needs to be changed.)
```

Comment:
sorry
what do you mean by "this" ?
does it mean "eth1" or "172.31.128.0/22" ?

Reply:
“this” means all the eth0 & eth1.  It refers [6.2.2.1.1. NICs](http://rackhd.readthedocs.io/en/latest/rackhd/ubuntu_package_installation.html)